.expeditor/scripts/verify-plan.ps1

```diff
@@ -25,6 +25,9 @@ if (-not ($source.name -match "git.exe")) {
     $env:Path = [System.Environment]::GetEnvironmentVariable("Path", "Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path", "User")
 }
 
+Write-Output "--- :construction: Verifying the Windows version we're running on"
```

Comment:
I like that we have a string that we can search for uniformly, could we also include that in omnibus-test.ps1 and omnibus-test.sh?